### DISCUSSION TITLE
Pull shark is not visible all of the sudden

### DISCUSSION THREAD

**Discussion by 21Ovi** [community/community#59335] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

I had pull shark x3 but now it dissapeared

**Comment by GhostOf0days** [comment, marked as answer]
Hello. Did you change your Github email address by chance? If so, please add it back to your Github account. It doesn't have to be the primary email address. Could you also please make sure you didn't accidentally hide that individual achievement (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements)?

**Reply by 21Ovi** [reply]
Thanks

**Comment by leonden** [comment]
Hi
Looks fine for me! I just went to your profile and the 3x Pull Shark was displayed as you said it should.

**Reply by 21Ovi** [reply]
Yup there was some confusion in email, thanks for the reply.
